druid/examples/blocking_function.rs

```diff
@@ -0,0 +1,100 @@
+use druid::ExtEventSink;
+use druid::Selector;
+
+use std::{thread, time};
+
+use druid::AppDelegate;
+use druid::DelegateCtx;
+use druid::Target;
```

Comment:
I'd like to have these imports ordered; imports from the same crate or module should be together (`druid::{a, bunch, of, stuff}`), `std` imports at the top, then crate imports.